apps/backoffice-v2/public/locales/en/toast.json

```diff
@@ -23,6 +23,10 @@
     "success": "The document has been asked for revision successfully.",
     "error": "Error occurred while asking for revision of the document."
   },
+  "revert_revision": {
+    "success": "Action cancelled successfully.",
```

Comment:
Please change to something indicative of which action was canceled, i.e. "Document ask to re-upload" or similar. (same for error)

Reply:
Done